domain/nsenter.go

```diff
@@ -132,6 +132,8 @@ type NSenterEventIface interface {
 	SetResponseMsg(m *NSenterMessage)
 	GetResponseMsg() *NSenterMessage
 	GetProcessID() uint32
+	ReapStartRequest()
```

Comment:
I don't see any callers for these new interface methods.

Reply:
Same as above.